protos/escrow/escrow.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+package btfs;
+
+service EscrowService{
+	rpc SubmitContracts(EscrowContractRequest) returns (EscrowContractResult) {}
+	rpc PayIn(PayinRquest) returns (PayinResult) {}
+	rpc CheckPayment(ContractID) returns (PayinStatus) {}
+	rpc CancelContracts(CancelContractRequest) returns (CancelContractResult) {}
+};
+
+enum TokenType {
+	BTT = 0;
+}
+
+enum Schedule {
+	MONTHLY = 0;
+	QUARTERLY = 1;
+	ANNUALLY = 2;
+}
+
+message ContractID {
+	bytes contract_id = 1;
+}
+
+message EscrowContract {
+	bytes contract_id = 1; // identity of a contract
+	bytes buyer_address = 2;
+	bytes buyer_signature = 3;
+	bytes seller_address = 4;
+	bytes seller_signature = 5; // the seller signature
+	bytes auth_address = 6; // authority who can cancel escrow
+	bytes auth_signature = 7;
```

Comment:
auth_signature cannot be provide at this time on the current procedure.

Reply:
so I will remove the auth_signature